benchmarks/dynamo/runner.py

```diff
@@ -264,6 +265,27 @@ def parse_args():
         default=DASHBOARD_DEFAULTS["dashboard_gh_cli_path"],
         help="Github CLI path",
     )
+    parser.add_argument(
+        "--batch_size", type=int, default=None, help="batch size for benchmarking"
+    )
+    parser.add_argument(
+        "--threads",
+        "-t",
+        type=int,
+        default=None,
+        help="number of threads to use for eager",
+    )
+    numactl_group = parser.add_argument_group("Numactl Parameters")
```

Comment:
Can we enable numactl options with a user opt-in flag? This keeps the default behavior.

Reply:
single_socket and single_core options are added.